### ISSUE TITLE
When will the CPU implementation of cuDNN be released?

### ISSUE THREAD

**Issue by barbaraIP** [Microsoft/CNTK#743]
Hi,

The CNTK library is used by our team to train & test a deep neural network with batch normalization. Training & testing work great on a Windows OS with a GPU (NVIDIA Quadro K1100M), but as soon as I want to run the testing process on a Windows OS without GPU it does not work anymore (there is no crash but the results do not make sense anymore).

Is this due to the fact that the CPU-only version of cuDNN is not yet implemented?
If yes, could you please tell me when it should be released?
I believe this is a quite urgent need...

Thanks in advance for your help!

**Comment by frankseide** [comment]
Some algorithms are indeed implemented through CUDA/cuDNN, and we don't have matching CPU algorithms. However, in these cases, you should get an error message.

Your situation sounds like a problem with the ACML library we had in an older version of CNTK. Please try to either upgrade to a latest version, or try this

    set ACML_FMA=0

See https://github.com/Microsoft/CNTK/wiki/Setup-CNTK-on-Windows#deprecated-acml

If it still does not work, please let us know.

**Comment by barbaraIP** [comment]
It works by replacing the ACML library with the MKL.
Thanks a lot!
